### ISSUE TITLE
Spec out "Measurable versioning"

### ISSUE THREAD

**Issue by OAGr** [foretold-app/foretold#534]
Every time a measurable is updated, the previous version should be saved. Later we can add a ui to show all of the versions of the measurable. This is important to make it clear if a name / description has changed in a significant way. 

Write out a few stories for doing this in the backend, and making these backups available via graphQL.

**Comment by RomanGalochkin** [comment]
One point to estimations and making stores.

**Comment by lagerros** [comment]
Would also be good if, at the time of making a measurements/prediction, the most recent measurable version was stored. 

For example:

Measurable: When will we colonise Mars? [v1]

Prediction: 10 to 100 years

_Measurable is edited_

Measurable: When will we colonise Mars with at least 3000 people? [v2]

Then we want to store that the prediction was made for v1, even though the most recent version is v2.

**Comment by OAGr** [comment]
I agree with you @lagerros , but want to keep this specific story to figuring out how the backend would work. The versioning should be relatively easy technically (more tricky in UX)

**Comment by RomanGalochkin** [comment]
Here is a doc (https://docs.google.com/document/d/1AEhRd9QuWO4O80PwGMXJHzPA_xRytAkxWm0WGAT0v38/edit?usp=sharing) I have created to support us to understand how to make it possible. 

So after this doc I understood that we could start from small steps increasing a functionality of the app with tiny portions. Going to add issues.

**Comment by OAGr** [comment]
Looks good, will spend time reviewing. Feel free to move on to the other issues.
